app/build.gradle

```diff
@@ -31,7 +31,7 @@ android {
     compileSdkVersion 29
     buildToolsVersion "29.0.2"
     defaultConfig {
-        applicationId "io.bluetrace.opentrace"
+        applicationId "com.kccreate.intelligentlockdown"
```

Comment:
the `.dev` suffix is commented, should we bring it back?
also, will this collide with @ChrisMTab task?

Reply:
Or you ok with this applicationId @ChrisMTab ?